fetcher.go

```diff
@@ -187,7 +187,7 @@ func (p *pool) fetchResource(ctx context.Context, from string, resource string,
 				// my address
 				Range:          "",
 				Referrer:       respReq.Referer(),
-				UserAgent:      respReq.UserAgent(),
+				UserAgent:      "bifrost-" + os.Getenv(EnvironmentKey),
```

Comment:
[the embedding bifrost-gatway](https://github.com/ipfs/bifrost-gateway/blob/3c65632495c8cc6e0db55103c0d9deea918204bb/http_client.go#L17) is already setting a useful version in the user agent. we shouldn't drop that.

Reply:
@willscott Okay, will append